frontend/src/pages/modelServing/screens/global/InferenceServiceListView.tsx

```diff
@@ -40,7 +45,10 @@ const InferenceServiceListView: React.FC<InferenceServiceListViewProps> = ({
   };
 
   const searchTypes = React.useMemo(
```

Comment:
Wow -- this is weird. We definitely should have done this differently. Might as well leave it for now.

Reply:
yeah i saw that... in an effort to not rip it all out and redo it. i just kept it